mobile.md

```diff
@@ -0,0 +1,35 @@
+# Mobile
+
+## Termin wykonania zadania:
+<!-- Dodaj termin wykonania zadania -->
+
+## Opis projektu
+Aplikacja mobilna wyświetlająca listę drinków wraz ze składnikami, zdjęciami oraz innymi szczegółami. Umożliwia przeglądanie drinków, ich filtrowanie oraz sortowanie.
+
+## API
+Do obsługi drinków wykorzystywane jest API bazujące na serwisie [TheCocktailDb](https://www.thecocktaildb.com/).
+- Dokumentacja API dostępna [tutaj](https://cocktails.solvro.pl/).
+
+## MVP (Minimum Viable Product)
```

Comment:
może dopisać że to funkcjonalności must to have (ale mvp też spoko więc niech zostanie to określenie)